qwandaq/src/main/java/life/genny/qwandaq/constants/FilterConst.java

```diff
@@ -0,0 +1,76 @@
+package life.genny.qwandaq.constants;
+
+public class FilterConst {
+    public static final String PAGINATION_INDEX="PRI_INDEX";
+    public static final String SEARCH_TEXT="PRI_SEARCH_TEXT";
+    public static final String  FILTER_LABEL = "FILTER_LABEL";
+
+    public static final String PRI_NAME="PRI_NAME";
+    public static final String PRI_CODE="PRI_CODE";
+    public static final String PRI_CREATED_DATE="PRI_CREATED_DATE";
+
+    /* BUCKET */
+    public static final String BUCKET_CODES = "BUCKET_CODES";
+    public static final String BUCKET_DISPLAY = "DISPLAY";
+    public static final String NONE = "NONE";
+    public static final String BUCKET_PROCESS = "PROCESS";
+
+    /* question */
+    public static final String QUE_QQQ_GROUP = "QQQ_QUESTION_GROUP";
```

Comment:
This is a duplicate of an existing code in `Attribute.java`

Reply:
Used attribute